song-server/src/main/java/bio/overture/song/server/controller/SchemaController.java

```diff
@@ -16,39 +16,64 @@
  */
 package bio.overture.song.server.controller;
 
-import bio.overture.song.server.model.dto.schema.GetSchemaResponse;
-import bio.overture.song.server.model.dto.schema.ListSchemaIdsResponse;
-import bio.overture.song.server.service.SchemaService;
+import bio.overture.song.server.model.dto.AnalysisType;
+import bio.overture.song.server.service.AnalysisTypeService;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
-import lombok.RequiredArgsConstructor;
+import lombok.NonNull;
+import lombok.SneakyThrows;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.Collection;
+
+import static bio.overture.song.core.utils.JsonUtils.readTree;
+import static bio.overture.song.core.utils.JsonUtils.toPrettyJson;
+
 @RestController
-@RequestMapping(path = "/schema")
-@RequiredArgsConstructor
+@RequestMapping(path = "/schemas")
```

Comment:
what's the point of having `/schemas` when you never call it without immediately following with `/analysis`.
Maybe you don't need to have `/schemas/analysis` and you can just stick with `/schemas/`.